packages/client/src/broadcast.ts

```
@@ -35,12 +35,19 @@ export interface WireCommand {
   content?: Blob;
 }
 
+/** A follower read intent: served by the leader's engine, answered by value. */
+export type WireRead =
+  | { kind: 'snapshot'; folder: Uint8Array }
+  | { kind: 'download'; node: Uint8Array };
```

ESLint

[error] Replace `|·{·kind:·'snapshot';·folder:·Uint8Array·}⏎·` with `{·kind:·'snapshot';·folder:·Uint8Array·}`

(prettier/prettier)